core/src/edu/asu/c3simulator/simulation/Employee.java

```diff
@@ -0,0 +1,248 @@
+		int randomLast = (int) (Math.random() * lastName.length);
+		
+		String randomEmployeeName = firstName[randomFirst] + " " + lastName[randomLast];
+		
+		return randomEmployeeName;
+	}
+	
+	public void setRandomEmployeeName()
+	{
+		this.employeeName = getRandomEmployeeName();
+	}
+	
+	/*
+	 * public String getEmployeePosition() { return employeePosition; }
+	 * 
+	 * public void setEmployeePosition(String employeePosition) { this.employeePosition =
+	 * employeePosition; }
+	 */
+	
+	public String getEmployeePosition()
+	{
+		String[] positions = { "Manager", "Product Design", "Sales Associate",
+				"Marketing" };
+		int randomPositions = (int) (Math.random() * positions.length);
+		employeePosition = positions[randomPositions];
+		return employeePosition;
+	}
+	
+	public void setEmployeePosition(String employeePosition)
+	{
+		this.employeePosition = employeePosition;
+	}
+	
+	public int getHourlyWage()
+	{
+		return employeePay;
+	}
+	
+	public void setHourlyWage(int employeePay)
+	{
+		this.employeePay = employeePay;
+	}
+	
+	/*
+	 * Employee Preference is determined by the selected employee's position and returns
+	 * an integer between the given pay range. The offset between the employee preference
+	 * and the employee pay determines the employee's morale.
+	 */
+	
+	public int getPreferredHourlyWage()
+	{
+		if (employeePosition == "Manager")
+		{
+			Random r = new Random();
+			int lowPay = 17;
+			int highPay = 24;
+			int i = r.nextInt(highPay - lowPay);
+			employeePref = i + lowPay;
+		}
+		
+		else if (employeePosition == "Product Design")
+		{
+			Random r = new Random();
+			int lowPay = 14;
+			int highPay = 19;
+			int i = r.nextInt(highPay - lowPay);
+			employeePref = i + lowPay;
+		}
+		
+		else if (employeePosition == "Sales Associate")
+		{
+			Random r = new Random();
+			int lowPay = 11;
+			int highPay = 16;
+			int i = r.nextInt(highPay - lowPay);
+			employeePref = i + lowPay;
+		}
+		
+		else if (employeePosition == "Marketing")
+		{
+			Random r = new Random();
+			int lowPay = 8;
+			int highPay = 13;
+			int i = r.nextInt(highPay - lowPay);
+			employeePref = i + lowPay;
+		}
+		
+		return employeePref;
+	}
+	
+	public void setEmployeePref(int employeePref)
+	{
+		this.employeePref = employeePref;
+	}
+	
+	/*
+	 * Employee Morale is determined by the difference between their pay preference and
+	 * their actual pay. If their morale reaches zero, the employee quits and is removed
+	 * from the active roster.
+	 */
+	
+	public int getMorale()
+	{
+		int employeeMorale = 10;
+		int payDifference = employeePref - employeePay;
+		
+		if (payDifference == 1)
+		{
+			employeeMorale -= (payDifference * 2);
+		}
+		
+		else if (payDifference == 2)
+		{
+			employeeMorale -= (payDifference * 2);
+		}
+		
+		else if (payDifference == 3)
+		{
+			employeeMorale -= (payDifference * 2);
+		}
+		
+		else if (payDifference == 4)
+		{
+			employeeMorale -= (payDifference * 2);
+		}
```

Comment:
Why is pay difference checked for values 1-4, when all blocks do the same thing?

if ( <=0)
else if (<= 4) // these numbers seem to be magic. Consider making them constants
else